### PR TITLE
Clean up unused variables and VIM directives.

### PR DESCRIPTION
**Pull request by tomohikokimura**

### CODE REVIEW

**Comment by tomohikokimura** [comment]
Hello Brandon;

Could you also review this pull request as well? It's just a clean-up in which I replaced unused variables for return values with '_', and unnecessary vim directives at the beginning of the files. I didn't touch the copy right statements per se as these sources have already been released under those and didn't see it appropriate to change them after the fact.

File: src/midonetclient/api.py [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# vim: tabstop=4 shiftwidth=4 softtabstop=4
-
 # Copyright 2013 Midokura PTE LTD.
```
Comment (brandon-berg) [inline src/midonetclient/api.py]: Fix copyright statement.

File: src/midonetclient/resource_base.py [hunk 2]
```diff
@@ -1,5 +1,3 @@
-# vim: tabstop=4 shiftwidth=4 softtabstop=4
-
 # Copyright 2013 Midokura PTE LTD.
```
Comment (brandon-berg) [inline src/midonetclient/resource_base.py]: Copyright.

File: src/midonetclient/resource_base.py [hunk 3]
```diff
@@ -80,19 +78,18 @@ def update(self, headers=None):
 
         self._ensure_content_type(headers)
 
-        resp, body = self.auth.do_request(self.dto['uri'], 'PUT',
-                                          body=self.dto,
-                                          headers=headers)
+        _, _ = self.auth.do_request(self.dto['uri'], 'PUT',
```
Comment (brandon-berg) [inline src/midonetclient/resource_base.py]: Do you really need "_, _ = "? Can't you just start the line with self.auth.do_request? Same with the last change in this file.

File: src/midonetclient/api.py (hunk 1 above)
Comment (tomohikokimura) [inline src/midonetclient/api.py]: Done.

File: src/midonetclient/resource_base.py (hunk 2 above)
Comment (tomohikokimura) [inline src/midonetclient/resource_base.py]: Done.

File: src/midonetclient/resource_base.py (hunk 3 above)
Comment (tomohikokimura) [inline src/midonetclient/resource_base.py]: A good point. Got rid of unused return values here and at another place.

**Comment by tomohikokimura** [comment]
Thanks, @brandon-berg . I've addressed your concern. Please take another look.

**Comment by brandon-berg** [comment]
Looks good now. Thanks!

**Pull request merged**